core/src/api/types.rs

```diff
@@ -501,7 +501,7 @@ impl TryFrom<RpcEvent> for Option<PublishMessage> {
 				.map(Box::new)
 				.map(PublishMessage::HeaderVerified)
 				.map(Some),
-			RpcEvent::ConnectedHost(_) => Ok(None), // silently skip ConnectedHost event
```

Comment:
Do we want catch all here or just list skipped events with `EventA | EventB => Ok(None)`?

Reply:
Ok, will go with the OR match, for the case of future developments